src/containers/SearchPage/FilterComponent.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-
-import { SCHEMA_TYPE_ENUM, SCHEMA_TYPE_MULTI_ENUM } from '../../util/types';
-import { constructQueryParamName } from './SearchPage.shared';
+import { SCHEMA_TYPE_ENUM, SCHEMA_TYPE_MULTI_ENUM, SCHEMA_TYPE_LONG } from '../../util/types';
```

Comment:
Actually, don't remove the empty lines. Instead, add comments around different imports.
Something like this:
https://github.com/sharetribe/web-template/blob/e16b014ae6461df71704e421c53f14e506869b1b/src/index.js#L36

(This was initiated after we started to receive issues that customizers didn't pay attention to imports and started to receive errors.)

So, we should probably give a bit more emphasis to the order of imports. I.e.  if imports are made in wrong bad order they might cause circular dependency issues. Therefore, it would be a good practice to add comments like `// utils`,  `// components` while touching old components.

Here's more about this problem scope:
https://github.com/sharetribe/web-template/blob/e16b014ae6461df71704e421c53f14e506869b1b/src/components/index.js#L5-L10